src/streamlink/plugins/neulion.py

```diff
@@ -0,0 +1,138 @@
+import re
+
+from streamlink.plugin import Plugin, PluginOptions
+from streamlink.plugin.api import http
+from streamlink.plugin.api import useragents
+from streamlink.plugin.api import validate
+from streamlink.stream import HLSStream
+from streamlink.utils import parse_json
+
+
+def js_to_json(data):
```

Comment:
just saw that this **js_to_json** can't handle some data with `:` or `,` in names were well.

it needs an update 🌊